documentation/modules/auxiliary/scanner/ssh/ssh_version.md

```diff
@@ -22,13 +37,219 @@ This module is tested on several different SSH services, such as:
 
 ### SSH-2.0 on GitHub
 
-  ```
-msf5 auxiliary(scanner/ssh/ssh_version) > use auxiliary/scanner/ssh/ssh_version
+```
+msf5 > use auxiliary/scanner/ssh/ssh_version
 msf5 auxiliary(scanner/ssh/ssh_version) > set RHOSTS github.com
 RHOSTS => github.com
 msf5 auxiliary(scanner/ssh/ssh_version) > run
 
-[+] 140.82.118.4:22       - SSH server version: SSH-2.0-babeld-38be96bc
-[*] github.com:22         - Scanned 1 of 1 hosts (100% complete)
+[*] 140.82.113.3 - Key Fingerprint: 65:96:2d:fc:e8:d5:a9:11:64:0c:0f:ea:00:6e:5b:bd
+[+] 140.82.113.3 - Host Key Encryption ecdsa-sha2-nistp256 uses a weak elliptic curve and should not be used.
+[*] 140.82.113.3 - SSH server version: SSH-2.0-babeld-8e18a363
+[*] 140.82.113.3 - Server Encryption
+=================
+
```

Comment:
I think this section needs to be updated again right? It looks like it's holding the old format.